Makefile

```diff
@@ -187,6 +186,11 @@ headers-check: ## Only check for problematic files.
 headers-write: ## Fix any problematic files blindly.
 	./hack/headers-write
 
+.PHONY: start
+start:
+	sudo $(HOME)/.cargo/bin/auraed
```

Comment:
Can you please use the `-E` flag with sudo? I know we aren't supposed to use sudo in Makefiles but its fine for now.

```
sudo -E auraed
```

I think `$PATH` should already include the cargo bin directory.